### PR TITLE
Added documentation for isos: attach/detach/extract

### PR DESCRIPTION
**Pull request by npaun**

### CODE REVIEW

File: source/includes/cloudstack/_isos.md [hunk 1]
```diff
@@ -106,3 +106,50 @@ Attributes | &nbsp;
...
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/isos/:id?operation=attach</code>
+
+Attach an existing, non-bootable ISO to an [instance](#cloudstack-instances). Each instance may have only one ISO attached at a time.
+
+Required                | &nbsp;                                                  
+------------------------|---------------------------------------------------------
+`instanceId`<br/>*UUID* | The id of the instance to which to attach the ISO to
```
Comment (pdube) [inline source/includes/cloudstack/_isos.md]: What about `to which the ISO will be attached` ?

File: source/includes/cloudstack/_isos.md [hunk 2]
```diff
@@ -106,3 +106,50 @@ Attributes | &nbsp;
...
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/compute-on/testing/isos/e922e5fc-8fee-4688-ad93-c9ef5d7eb685?operation=generateDownloadUrl"
+
+# No parameters required
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/isos/:id?operation=generateDownloadUrl</code>
+
+Generate a URL that can be used to download an ISO. The "Allow generation of download URL" (`extractable`) option must have been selected when the ISO was imported. The download URL will be placed in the `url` field of the task result.
```
Comment (pdube) [inline source/includes/cloudstack/_isos.md]: The second sentence might be a bit too UI oriented. I also added another possible use case for the generate url.  `Generate a URL that can be used to download an extractable ISO. The URL may also be used to import the ISO into another region. To be able to generate the download URL, the ISO must have been imported with the extractable flag set to true. The download URL will be placed in the url field of the task result.`

**Review by pdube — APPROVED** [review]

File: source/includes/cloudstack/_isos.md (hunk 1 above)
Comment (npaun) [reply source/includes/cloudstack/_isos.md]: Better; changing

File: source/includes/cloudstack/_isos.md (hunk 2 above)
Comment (npaun) [reply source/includes/cloudstack/_isos.md]: Thank you, fixed.

**Review by simongodard — APPROVED** [review]
LGTM !

File: source/includes/cloudstack/_isos.md [hunk 3]
```diff
@@ -115,7 +115,7 @@ curl -X POST \
    -H "MC-Api-Key: your_api_key" \
    -d "request_body" \
    "https://cloudmc_endpoint/v1/services/compute-on/test_area/isos"
-```
```
Comment (pdube) [inline source/includes/cloudstack/_isos.md:118]: Is this a mistake?

File: source/includes/cloudstack/_instances.md [hunk 4]
```diff
@@ -504,3 +504,17 @@ Associate a new [SSH key](#cloudstack-ssh-keys) to the default user of an existi
...
 `sshKeyName`<br/>*string* | The name of the [SSH key](#cloudstack-ssh-keys) to associate to the instance
+
+#### Detach the ISO from an instance
+
+```shell
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/compute-on/testing/instances/e922e5fc-8fee-4688-ad93-c9ef5d7eb685?operation=detachIso"
+
```
Comment (pdube) [inline source/includes/cloudstack/_instances.md:515]: Is this missing closing ``` ?

**Pull request merged**